src/components/header/header.tsx

```diff
@@ -0,0 +1,9 @@
+import { StyledHeader } from "./header.styled";
+
+export const Header = () =>
+    <StyledHeader>
```

Comment:
formatting goes a long way :)
Maybe you should properly define auto formatting, linting and organize imports on save?

Reply:
Prettier and lint added.